app/src/main/kotlin/at/bitfire/davdroid/sync/ContactsSyncManager.kt

```diff
@@ -145,23 +143,12 @@ class ContactsSyncManager @AssistedInject constructor(
     private lateinit var resourceDownloader: ResourceDownloader
 
 
-    override fun prepare(): Boolean {
-        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O) {
-            // workaround for Android 7 which sets DIRTY flag when only meta-data is changed
-            val reallyDirty = localCollection.verifyDirty()
-            val deleted = localCollection.findDeleted().size
-            if (extras.contains(ContentResolver.SYNC_EXTRAS_UPLOAD) && reallyDirty == 0 && deleted == 0) {
-                logger.info("This sync was called to up-sync dirty/deleted contacts, but no contacts have been changed")
-                return false
-            }
-        }
-
```

Comment:
I'm a bit confused – shouldn't that be done in #1090, too?

This PR should only update vcard4android so that the `account` fields are renamed to `addressBookAccount`.

Reply:
> I'm a bit confused – shouldn't that be done in https://github.com/bitfireAT/davx5-ose/pull/1090, too?

Yup, somehow I mixed the two PRs, I'll clean up this mess 😅